nexmo/__init__.py

```diff
@@ -166,9 +166,17 @@ def control_verification_request(self, params=None, **kwargs):
   def get_basic_number_insight(self, params=None, **kwargs):
     return self.get(self.api_host, '/number/format/json', params or kwargs)
 
+  def get_standard_number_insight(self, params=None, **kwargs):
+    return self.get(self.api_host, '/number/lookup/json', params or kwargs)
```

Comment:
@judy2k Not a mistake, more just confusion as to when the endpoints were changing, and whether that was a backwards compatible change or not.

The get_standard_number_insight method should presumably point to `/ni/standard/json`, and get_basic_number_insight to `/ni/basic/json`?